### DISCUSSION TITLE
RRF3 support

### DISCUSSION THREAD

**Discussion by goldjunge91** [OctoFarm/OctoFarm#1214] [Feature Request]
Hi, first I would say very very crazy good work on this.
And my feature request is supporting Klipper as we’ll RRF3 it would grate to combine all my printer to one interface upload and track up and down time.

**Comment by NotExpectedYet** [comment]
I noticed you'd seen the other threads but there is a lot of de-coupling and re-working of the code needed to support this. 

One day! I'm writing V2.0 with supporting multiple interfaces in mind though. 

Just to not, the Klipper request is already here so I've renamed this for RRF.

**Comment by NotExpectedYet** [comment]
Please see: https://github.com/OctoFarm/OctoFarm/discussions/1298
